packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -434,6 +489,46 @@ enum BillingResponse {
   networkError,
 }
 
+/// Plugin concept to cover billing modes.
+///
+/// [playBillingOnly] (google play billing only).
+/// [alternativeBillingOnly] (app provided billing with reporting to play).
```

Comment:
Leave a comment here saying the values must match the BillingChoiceMode enum on the Java side?

Reply:
Good idea done.